src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.SymbolMoniker.cs

```diff
@@ -21,8 +21,7 @@ internal abstract partial class AbstractFindUsagesService
         private static async Task FindSymbolMonikerReferencesAsync(
             IFindSymbolMonikerUsagesService monikerUsagesService,
             ISymbol definition,
-            IFindUsagesContext context,
-            CancellationToken cancellationToken)
+            IFindUsagesContext context)
```

Comment:
context and cancellationToken together are redundant (since context has the cancellationToken).  I removed the latter, esp. so we don't accidentally pass the wrong one along.